hivemind-host/proto/message.proto

```diff
@@ -1,9 +1,63 @@
 syntax = "proto3";
-import "generic.proto";
-import "user-call.proto";
 
 option java_package = "com.swarmus.hivear";
 
+// Generic
+enum GenericResponseStatus {
+    OK = 0;
+    BAD_REQUEST = 1;
+    ERROR = 2;
+}
+
+message GenericResponse {
+    GenericResponseStatus status = 1;
+    string details = 2;
+}
+
+// FunctionCall
+message FunctionArgument {
```

Comment:
is it possible to add an optional argument name here?
Would be nice to retrieve what is the argument about